bundles/automation/org.eclipse.smarthome.automation.json.parsers/pom.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <parent>
+    <groupId>org.eclipse.smarthome.bundles</groupId>
+    <artifactId>automation</artifactId>
+    <version>0.8.0-SNAPSHOT</version>
+  </parent>
+
+  <properties>
+    <bundle.symbolicName>org.eclipse.smarthome.automation.json.parsers</bundle.symbolicName>
+    <bundle.namespace>org.eclipse.smarthome.automation.json.parsers</bundle.namespace>
+  </properties>
+
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.eclipse.smarthome.automation</groupId>
+  <artifactId>org.eclipse.smarthome.automation.json.parsers</artifactId>
+
+  <name>Eclipse SmartHome Automation JSON Parsers</name>
+
+  <packaging>eclipse-plugin</packaging>
+
+	<dependencies>
+		<dependency>
```

Comment:
Hi Kai,
the Config class seems to have been changed since my last update and my sources became outdated when I synched just before the commit, now it is ok - I have updated the source to the new constructor
I have updated also the classpath for the second issue.
Regarding the json dependency please check the target platform file in the commit, I tried to use both the plugin from orbit repo and from maven.org but with no success. Please check if this is the right place to update the target platform.
One important note: the normal org.json plugin doesnot have export package header in the manifest file, so I am using the org.json-osgi plugin which has this header, it is with groupId
de.twentyeleven.skysail
Best regards,
Marin